hooks/build

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+# this script is used to build a Radicale Docker image locally
+# as well as on the Docker Hub.
+
+echo "------ HOOK START - BUILD -------"
+
+BASE_IMAGE=$(head -n1 Dockerfile | cut -d" " -f2)
+docker pull $BASE_IMAGE
```

Comment:
Manual `docker pull` is not needed. `docker build` does it automatically. And if you need to update it all the time, you can use `docker build --pull`.

Reply:
yeah, that script was copied from another project where i build the image for one architecture.